core/src/main/java/oracle/weblogic/deploy/util/WLSDeployArchive.java

```diff
@@ -62,7 +62,7 @@ public class WLSDeployArchive {
      * Top-level archive subdirectory where the applications are stored and the subdirectory to which
      * they will be extracted. This is for structured applications found under /app
      */
-    public static final String ARCHIVE_APPS_FOLD_TARGET_DIR = WLSDPLY_ARCHIVE_BINARY_DIR + "/application_folder";
+    public static final String ARCHIVE_APPS_FOLD_TARGET_DIR = WLSDPLY_ARCHIVE_BINARY_DIR + "/applicationsFolder";
```

Comment:
I hate to be a pain but please change this name to `applicationFolders` to more accurately reflect the purpose/content.  That is, this is a location for structured application folders not a single folder for applications.

Reply:
I think `structuredApplications` is clearer for the user.  Sounds more intuitive as to why we separated the two application types.